tools/acle_template.tex

```diff
@@ -0,0 +1,127 @@
+% Copyright (c) 2014--2021, John MacFarlane
+% Based on default.latex template released with Pandoc (www.pandoc.org).
+% License: BSD 3-clause
+% See `data/templates` in `COPYRIGHT` at https://github.com/jgm/pandoc
+
+\documentclass[$if(fontsize)$$fontsize$,$endif$$if(papersize)$$papersize$paper,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
```

Comment:
@ValeriaDaneva  - one more question. Have you modified this template at all after importing it from pandoc? Or did you configure the PDF output only via the yaml file used as a header?